lib/web/mage/menu.js

```diff
@@ -440,6 +440,12 @@ define([
                     if (!target.hasClass('level-top') || !target.has('.ui-menu').length) {
                         window.location.href = target.find('> a').attr('href');
                     }
+
+                    submenu = target.find('ul.submenu').first();
+
+                    if (submenu.attr('aria-expanded') == "true") {
```

Comment:
Hi @fmineo, I am not sure about the origin of the static test failure on your fix, but could you please try this suggestion and see if the test will pass.

```suggestion
                    if (submenu.attr('aria-expanded') === 'true') {
```

Reply:
Also please follow the recommendation of @manavluhar and update your branch, it is quite behind the mainline at the moment.